pkgs/applications/misc/cpp-ethereum/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
     sha256 = "1fxgpqhmjhpv0zzs1m3yf9h8mh25dqpa7pmcfy7f9qiqpfdr4zq9";
   };
 
-  cmakeFlags = with stdenv.lib; concatStringsSep " " ([ "-DCMAKE_BUILD_TYPE=Release" ] ++ extraCmakeFlags);
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" extraCmakeFlags ];
```

Comment:
I see... `CMAKE_BUILD_TYPE=Release` is [default in Nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L52), unless you use a [custom `configurePhase`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L61-L64).